ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/MaxLineLengthRuleTest.kt

```diff
@@ -90,6 +90,41 @@ class MaxLineLengthRuleTest {
         )
     }
 
+    @Test
+    fun testErrorSuppressionOnString() {
+        val testFile = ignoreString()
+
+        assertThat(
+            MaxLineLengthRule().lint(
+                testFile.absolutePath,
+                """
+                fun main() {
+                    val longString = "this is a very very very very long long string on a single line"
+                }
+                """.trimIndent(),
+                userData = mapOf("max_line_length" to "40")
+            )
+        ).isEmpty()
+    }
+
```

Comment:
Please add a test in which both properties are set. Experiment with a test with following scenario:
```
val `backTickedProperty` = "some very very very very long string on a single line"
```